workflow/lib/preprocess/file_utils.py

```diff
@@ -34,18 +32,3 @@ def get_sample_fps(
 
     # Return the list of file paths as Path objects
     return filtered_df["sample_fp"].tolist()
```

Comment:
I got a comment from Sarah, who I've connected you to, that the discrete file naming approach is a pain for instances where the file structure changes. So this approach will be much better.